### PR TITLE
Custom props

### PR DESCRIPTION
**Pull request by jyash97**
Usage

```js
customProps={{
	ReactiveList: {
		onData: res => (
			<div style={{ background: 'aqua' }}>{JSON.stringify(res)}</div>
		),
		style: { background: 'red' },
	},
	DataSearch: {
		renderSuggestions: res => (
			<div style={{ background: 'yellow' }}>{JSON.stringify(res)}</div>
		),
	},
}}
```

### CODE REVIEW

File: components/SearchSandbox/components/RSWrapper.js [hunk 1]
```diff
@@ -354,8 +359,7 @@ export default class RSWrapper extends Component {
 
 	renderFormItem = (item, name) => {
 		let FormInput = null;
-		const value =
-			this.props.componentProps[name] !== undefined
+		const value =			this.props.componentProps[name] !== undefined
```
Comment (metagrover) [inline components/SearchSandbox/components/RSWrapper.js]: Linting issue

File: components/SearchSandbox/containers/Editor.js [hunk 2]
```diff
@@ -379,12 +379,13 @@ export default class Editor extends Component {
 			if (type === 'string' || type === 'number') {
 				return (
 					<TreeNode
-						title={
-							<div>
-								<span>{item}:</span>&nbsp;
+						title={(
+<div>
```
Comment (metagrover) [inline components/SearchSandbox/containers/Editor.js]: Linting issue

File: components/SearchSandbox/containers/Editor.js [hunk 3]
```diff
@@ -447,10 +447,12 @@ export default class Editor extends Component {
 					<pre style={{ width: 300 }}>{JSON.stringify(res, null, 4)}</pre>
 				)
 			}
-			title={
-				<Row>
+			title={(
+<Row>
```
Comment (metagrover) [inline components/SearchSandbox/containers/Editor.js]: Linting issue

**Comment by jyash97** [comment]
Will create a separate PR to resolve all linting issues of Batteries

**Pull request merged**